resources/cmake/FindRTIConnextDDS.cmake

```diff
@@ -264,6 +264,31 @@
 # - All the Darwin platforms (OS X 10.11-10.13)
 #
 
+#####################################################################
+# Logging Functions                                                  #
+#####################################################################
+function(connextdds_debug message)
+    message(DEBUG ${message})
+endfunction()
+
+function(connextdds_verbose message)
+    message(VERBOSE ${message})
```

Comment:
You could rename these functions to:
connextdds_log_verbose
connextdds_log_debug

or

connextdds_message_verbose,
connextdds_message_debug

to give a context about what they do.

Reply:
I would go with `connextdds_log_xxxx` to match the function below (`connextdds_log_xml`)